README.md

```diff
@@ -31,45 +22,50 @@ Technical documentation specific to the Sinopia BIBFRAME Editor may also be foun
 2.  Install [npm](https://www.npmjs.com/)
 3.  Run `npm init`, and follow the instructions that appear.
 4.  Get latest npm: `npm install -g npm@latest`
-5.  Run `npm install grunt-cli` to install grunt-cli.
-6.  Run `npm install`. This installs everything needed for the build to run successfully.
-7.  Run `npm run build` to build the code.  (TEMP: Run `npm run grunt` or `grunt` to build the BFE code.)
+5.  Run `npm run build` to build the code.
+
+### Installing BFE code
+1.  Run `npm install grunt-cli` to install grunt-cli.
+2.  Run `npm run grunt` or `grunt` to build the BFE code.
+3.  Run `npm install`. This installs everything needed for the build to run successfully.
+
 
 ## Running the code
 
 `npm start`
 
 Follow installation instructions, then run `npm start` or `node server.js` to start the web server using Express.
-This will start up the code at http://localhost:8000.
-
-TEMP: The Sinopia Editor code is currently available via http://localhost:8000/bfe-index.html.
+This will start up the code at [http://localhost:8000](http://localhost:8000).
 
-TEMP:  Note that `index.html` is meant to show the Sinopia home page via React.
+The Sinopia Editor code is currently available via [stage.sinopia.io][staging]
```

Comment:
ah ha!   Should this be with the other reference to it above?  Or not?

Reply:
I adjusted the reference in earlier section.